enterprise/cmd/llm-proxy/internal/actor/actor.go

```diff
@@ -85,8 +85,8 @@ type updateOnFailureLimiter struct {
 	*Actor
 }
 
-func (u updateOnFailureLimiter) TryAcquire(ctx context.Context) error {
-	err := (limiter.StaticLimiter{
+func (u updateOnFailureLimiter) TryAcquire(ctx context.Context) (func() error, error) {
```

Comment:
```suggestion
func (u updateOnFailureLimiter) TryAcquire(ctx context.Context) (commit func() error, _ error) {
```
for docs purposes, might make it easier to understand what it's doing from the callsite?

Reply:
The return parameters are named on the interface itself, and it's named appropriately below from the "inner" limiter - I try to avoid named returns except for err capture due to gotchas